scipy/sparse/csgraph/_shortest_path.pyx

```diff
@@ -1536,7 +1527,7 @@ ctypedef FibonacciNode* pFibonacciNode
 
 
 cdef struct FibonacciHeap:
-    FibonacciNode* min_node
+    FibonacciNode* min_node # min_node.left_sibling is always NULL
     pFibonacciNode[100] roots_by_rank  # maximum number of nodes is ~2^100.
```

Comment:
The number of roots (equals to `100` here and in other place) might better be [defined as a literal value via a Cython `DEF` statement](https://cython.readthedocs.io/en/latest/src/userguide/language_basics.html?highlight=%22DEF%22#compile-time-definitions):

```cython
DEF N_ROOTS = 100
```

and replace the hard-coded value.

Reply:
Thanks, I'll replace it.